app/assets/javascripts/practice_editor_overview.es6

```diff
@@ -2,11 +2,17 @@
     const $document = $(document);
     const CHARACTER_COUNTER_INVALID_COLOR = '#e52207';
     const CHARACTER_COUNTER_VALID_COLOR =  '#a9aeb1';
-    
+
     const NAME_CHARACTER_COUNT = 50;
     const TAGLINE_CHARACTER_COUNT = 150;
     const SUMMARY_CHARACTER_COUNT = 400;
 
+    let cropper;
```

Comment:
⚠️ ideally, i would love a modular way to implement the cropper without having to add the following lines of code to each and every page with a cropper... 🤔

Reply:
we can probably throw it in the `utilities` file, then?

Reply:
`app/assets/javascripts/_practice_editor_utilities.es6`

Reply:
done! thank you for your guidance on this!